pubsub/gochannel/pubsub.go

```diff
@@ -26,6 +30,10 @@ type Config struct {
 	// When true, Publish will block until subscriber Ack's the message.
 	// If there are no subscribers, Publish will not block (also when Persistent is true).
 	BlockPublishUntilSubscriberAck bool
+
+	// When true, messages sent to a topic without any subscribers will be sent to the
+	// subscribers of the `*` topic.
+	EnableFallback bool
```

Comment:
maybe it would be a bit longer, IMO it could be a bit more explicit, like `EnableNoSubscribersFallback`

Reply:
Yup, I agree that the longer version is more expressive and changed it.